src/src_V2/modules/manager/supplier-management/accommodation/list/MultipleSelectCheckmarks.jsx

```diff
@@ -1,11 +1,11 @@
-import * as React from 'react';
-import OutlinedInput from '@mui/material/OutlinedInput';
-import InputLabel from '@mui/material/InputLabel';
-import MenuItem from '@mui/material/MenuItem';
+import Checkbox from '@mui/material/Checkbox';
 import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
 import ListItemText from '@mui/material/ListItemText';
+import MenuItem from '@mui/material/MenuItem';
+import OutlinedInput from '@mui/material/OutlinedInput';
 import Select from '@mui/material/Select';
-import Checkbox from '@mui/material/Checkbox';
+import * as React from 'react';
```

Comment:
please remove this import

Reply:
import * as React from 'react'; was previously there. This is the change of `importOrderSortSpecifiers`. Will change it accordingly.